content/docs/user-guide/project-structure/dvcyaml-files.md

```diff
@@ -76,34 +61,88 @@ Refer to [Visualizing Plots] and `dvc plots show` for more examples.
 
 ### Available configuration fields
 
-- `y` - source from which the Y axis data comes from:
+- `y` - source for the Y axis data:
+
+  - **Top-level plots** (_string, list, dict_):
 
-  - Top-level plots: accepts string, list, or dictionary (like
-    `data_source_path: column/field name`).
+    If plot ID is a path, one or more column/field names is expected. For
+    example:
 
-  - Plot outputs: column/field name found in the source plots file.
+    ```yaml
+    plots:
+      - regression_hist.csv:
+          y: mean_squared_error
+      - classifier_hist.csv:
+          y: [acc, loss]
+    ```
 
-- `x` (string) - source from which the X axis data comes from. An auto-generated
-  _step_ field is used by default.
+    If plot ID is an arbitrary string, a dictionary of file paths mapped to
+    column/field names is expected. For example:
 
-  - Top-level plots: multiple `x` values are supported, but only if they match
-    the number of `y` values and are specified as a dictionary (list is not
-    supported).
+    ```yaml
+    plots:
+      - train_val_test:
+          y:
+            train.csv: [train_acc, val_acc]
+            test.csv: test_acc
+    ```
 
-  - Plot outputs: column/field name found in the source plots file.
+  - **Plot outputs** (_string_): one column/field name.
 
-- `y_label` (string) - Y axis label. If all `y` data sources have the same field
-  name, that will be the default. Otherwise, it's "y".
+- `x` - source for the X axis data. An auto-generated _step_ field is used by
+  default.
 
-- `x_label` (string) - X axis label. If all `y` data sources have the same field
-  name, that will be the default. Otherwise, it's "x".
+  - **Top-level plots** (_string, dict_):
 
-- `title` (string) - header for the plot(s). Defaults:
+    If plot ID is a path, one column/field name is expected. For example:
 
-  - Top-level plots: `path/to/dvc.yaml::plot_id`
-  - Plot outputs: `path/to/data.csv`
+    ```yaml
+    plots:
+      - classifier_hist.csv:
+          y: [acc, loss]
+          x: epoch
+    ```
 
-- `template` (string) - [plot template]. Defaults to `linear`.
+    If plot ID is an arbitrary string, `x` may either be one column/field name,
+    or a dictionary of file paths each mapped to one column/field name (the
+    number of column/field names must match the number in `y`).
+
+    ```yaml
+    plots:
+      - train_val_test: # single x
+          y:
+            train.csv: [train_acc, val_acc]
+            test.csv: test_acc
+          x: epoch
+      - roc_vs_prc: # x dict
+          y:
+            precision_recall.json: precision
```

Comment:
This example is pretty long. Maybe we can summarize? And/or collapse the entire Plots/[Available fields](https://dvc.org/doc/user-guide/project-structure/dvcyaml-files#available-configuration-fields) section (with `<details>`).